website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -417,11 +423,32 @@ Indexing is one of the key features of ComposeDB. In order to notify the Ceramic
 composedb did:from-private-key your-private-key
 ```
 
-⚠️ Copy this authenticated DID key and store it in a secure place, just like with your private key above. This DID key will have to be provided in your Ceramic node’s configuration file which will ensure that only authorized users can make changes to your application, e.g. deploy models on your Ceramic node.
+The generated DID key will have the following format:
+
+```json
+did:key:...
+```
+
+:::caution
+
+Copy this authenticated DID key and store it in a secure place, just like with your private key above. This DID key will have to be provided in your Ceramic node’s configuration file which will ensure that only authorized users can make changes to your application, e.g. deploy models on your Ceramic node.
+
+:::
 
 ### Using your account
 
-Provide the authenticated DID by opening the node configuration file which defaults to `~/.ceramic/daemon.config.json` and specifying it in the `admin-dids` section of the file as shown below:
+The very first time you spin up a Ceramic node, a node configuration file is automatically created for you where you can configure how your node is operated. Here you have to provide the DID key which is authorised to interact with the Admin API.
+The Ceramic node configuration file will be created inside of the automatically created directory `./ceramic` which can be access using the following command:
```

Comment:
While "~/.ceramic" should be clear that it's in the user's home directory, nobody knew this during user testing.

So maybe let's make it even more explicit?

---

The Ceramic node configuration file will be created inside of the automatically created directory `./ceramic` in your home directory (usually `/home/USERNAME/` on Linux or `/Users/USERNAME/` on Mac)

This directory can be accessed using the following command:

Reply:
Great suggestion! Added.